pkg/apis/resources/v1alpha1/types.go

```diff
@@ -195,6 +195,10 @@ const (
 	// it's updateMode field option should not be modified to "InPlaceOrRecreate"
 	// TODO(vitanovs): Remove the VPAInPlaceUpdatesSkip label constant once the VPAInPlaceUpdates feature gates is deprecated.
 	VPAInPlaceUpdatesSkip = "vpa-in-place-updates.resources.gardener.cloud/skip"
+	// VPAInPlaceUpdatesMutated is a constant for a label on a VerticalPodAutoscaler resource which indicates that
+	// it's updateMode field has already been mutated.
+	// TODO(vitanovs): Remove the VPAInPlaceUpdatesMutated label constant once the VPAInPlaceUpdates feature gates is deprecated.
```

Comment:
We don't want to deprecate the feature gate, we want to promote it to GA.

```suggestion
	// TODO(vitanovs): Remove the VPAInPlaceUpdatesMutated label constant once the VPAInPlaceUpdates feature gates is promoted to GA.
```

Reply:
This comment is not yet addressed.

Updated suggestion:
```suggestion
	// TODO(vitanovs): Remove the VPAInPlaceUpdatesMutated label constant once the VPAInPlaceUpdates feature gate is promoted to GA.
```